lib/databaseOps.js

```diff
@@ -0,0 +1,39 @@
+var mysql = require("mysql2");
+
+module.exports = {
+  connectToDB: () => {
+    var con = mysql.createConnection({
+      host: process.env.MYSQL_HOSTNAME,
+      user: process.env.MYSQL_USER,
+      password: process.env.MYSQL_PASSWORD,
+      database: process.env.MYSQL_DATABASE,
+    });
+
+    con.connect(function (err) {
+      if (err) throw err;
```

Comment:
this error is actually going to be thrown in the context of the anonymous function, it won't be surfaced to the parent function.

we'll want the connection here to be asynchronous, so `connectToDB` can fail if we can't connect to the database.  take a look at https://www.npmjs.com/package/mysql2#using-promise-wrapper to get started

Reply:
I believe that I fixed it with the new commit.